logger.go

```diff
@@ -0,0 +1,50 @@
+package logger
+
+import (
+	"os"
+
+	"github.com/evalphobia/logrus_sentry"
+	"github.com/sirupsen/logrus"
+)
+
+var logger *logrus.Logger
+
+// F is a map used to log custom fields.
+type F map[string]interface{}
+
+//Init initializes the logger to be used in any package.
+func init() {
+	logger = logrus.New()
+
+	dsn, exists := os.LookupEnv("SENTRY_DSN")
+
+	if exists {
+		hook, err := logrus_sentry.NewSentryHook(dsn, []logrus.Level{
+			logrus.PanicLevel,
+			logrus.FatalLevel,
+			logrus.ErrorLevel,
+		})
+		if err != nil {
+			logger.Warnf("It was not possible to enable Sentry handler: %w", err)
```

Comment:
Is it simply a warning? Is the function supposed to continue normally after this?